dds/DCPS/MulticastManager.cpp

```diff
@@ -165,14 +172,18 @@ bool MulticastManager::process(InternalDataReader<NetworkInterfaceAddress>::Samp
     }
   }
 
-  if (failed_a_join && !any_joined && log_level >= LogLevel::Error) {
-    ACE_ERROR((LM_ERROR, "(%P|%t) ERROR: MulticastManager::process: "
```

Comment:
It looks like we don't have any actual error messages anymore, only warnings. I think we wanted the error messages to be able to have the test framework catch / report when these problems show up.

Reply:
Okay.  I'll upgrade the join/leave failures to errors.